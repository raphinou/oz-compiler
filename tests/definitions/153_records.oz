% Records where values are not constants
local
   R A B R2
in
   A=1
   B=2
   R=rec(a:A b:B )
   R2=rec(a r:R)
   {Show R}
   {Show R2}
end
