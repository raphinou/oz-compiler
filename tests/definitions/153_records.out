rec(a:1 b:2)
rec(1:a r:rec(a:1 b:2))
